lib/display/display.ex

```diff
@@ -19,13 +19,18 @@ defmodule Inky.Display do
 
   @spec spec_for(:impression) :: Inky.Display.t()
   def spec_for(type = :impression) do
+    width = 600
+    height = 448
```

Comment:
I experimented with the width and heights passed here and was able to get the display to show at 1/2 size by doubling them. The display crashed when they were quadrupled. There must be another way...